enterprise/frontend/src/embedding-sdk/test/auth-flow.unit.spec.tsx

```diff
@@ -187,4 +194,27 @@ describe("SDK auth flow", () => {
       });
     });
   });
+
+  describe("locale", () => {
```

Comment:
This test is needed because if we wrap https://github.com/metabase/metabase/pull/50078/files#diff-a0d9a5a0fd9f7a2bd796541f866d84d5ff1b76a4709c3ac31cb1a0fa7db8d3d1R37-R39
in a use effect, we encounter this issue where we try to load the locale from the current domain instead of from metabase